crates/cxx-qt-gen/src/generator/rust/externcxxqt.rs

```diff
@@ -21,15 +21,58 @@ impl GeneratedRustFragment {
     ) -> Result<Self> {
         let mut generated = Self::default();
 
+        let namespace = if let Some(namespace) = &extern_cxxqt_block.namespace {
```

Comment:
could call this `extern_block_namespace` to avoid any confusion / shadowing in the `map` where that has a namespace too

Reply:
Done